arm-web/2016-06-01/swagger/connectionGateways.json

```diff
@@ -277,11 +295,14 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/locations/{location}/Microsoft.Web/connectionGatewayInstallations/{gatewayInstallationName}": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/connectionGatewayInstallations/{gatewayInstallationName}": {
```

Comment:
was this URL incorrect? are there test cases validating for this spec? are those passing?